manifests/02-role.yaml

```diff
@@ -22,6 +29,14 @@ rules:
   - configmaps
   - secrets
   - serviceaccounts
+  - namespaces
```

Comment:
do we actually need the namespace permissions? seems overkill

Reply:
I probably added that during the E2E test debugging. Fixed.